frontend/tests/unit/specs/wells/__snapshots__/WellDetail.spec.js.snap

```diff
@@ -229,13 +229,7 @@ exports[`WellDetail.vue should match snapshot 1`] = `
               Aquifer Number: 
             </span>
              
-            <router-link-stub
-              to="[object Object]"
-            >
-              
-              
-            
-            </router-link-stub>
+            <!---->
```

Comment:
What were the changes made to this file?

Reply:
this was from running tests:update the snapshot of the page has changed since it was last run?